test/protos/instances.go

```diff
@@ -36,15 +36,15 @@ func GetNginxOssInstance() *v1.Instance {
 					StubStatus:      "",
 					AccessLogs:      []string{},
 					ErrorLogs:       []string{},
-					LoadableModules: []string{},
+					LoadableModules: nil,
 					DynamicModules:  []string{},
 				},
 			},
 		},
 	}
 }
 
-func GetNginxPlusInstance() *v1.Instance {
+func GetNginxPlusInstance(loadableModules string) *v1.Instance {
```

Comment:
Should the `GetNginxOssInstance` function also be updated to take a loadableModules param as well to be consistent?

Reply:
I was thinking more about this and as we add more information I'll need it to take more params so I'm not sure if this is the best way to do this or should I be passing in the entire runtime info

Reply:
Changed GetNginxOssInstance to take loadableModules param can see what to do about more params later